src/widgets/mobile-footer.tsx

```
@@ -0,0 +1,33 @@
+import Link from "next/link";
+import { Archive, BookOpen, Clock8, History } from "lucide-react";
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)